crates/networking/rpc/rpc.rs

```diff
@@ -76,19 +77,43 @@ trait RpcHandler: Sized {
     fn handle(&self, storage: Store) -> Result<Value, RpcErr>;
 }
 
+const FILTER_DURATION: Duration = {
+    if cfg!(test) {
```

Comment:
Didn't know about the `cfg!` macro, I like that way of solving the issue for testing